### ISSUE TITLE
How to do mixed workload benchmarking

### ISSUE THREAD

**Issue by sirizhou** [minio/warp#161]
May I start a warp process which includes two thread, one of them benchmark put operation while another one benchmark list operation.
## Existing method：
I start a warp process which benchmarks put operation then start another warp process benchmark list operation at the same time.
## Desired method:
benchmark mixed workload at one warp process

**Comment by harshavardhana** [comment]
`warp mixed` ?

**Comment by sirizhou** [comment]
Did I misunderstand the meaning of warp mixed？ _warp mixed_ benchmark get, stat, put, delete operation in turn ?
get ---->  stat ----> put ---> delete
or 
put->get->stat->delete->put->get->stat->delete--->....      just mixed the operation
(My English is poor， please forgive me)

**Comment by sirizhou** [comment]
I want the I/O queue in storage is r-w-r-r-w-w-r-w

**Comment by harshavardhana** [comment]
@sirizhou `warp` does mixed https://github.com/minio/warp#mixed automatically in parallel 

> I want the I/O queue in storage is r-w-r-r-w-w-r-w

what you are asking is out of scope for this project
